specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/examples/VerifyHostingEnvironmentVnet.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "api-version": "2020-09-01",
+    "parameters": {
+      "properties": {
+        "vnetResourceGroup": "vNet123rg",
+        "vnetName": "vNet123",
+        "vnetSubnetName": "vNet123SubNet"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/providers/Microsoft.Web/verifyHostingEnvironmentVnet",
```

Comment:
Please provide examples for all APIs exposed by 2020-09-01 including one for the new /providers/Microsoft.Web/generateGithubAccessTokenForAppserviceCLI API.

Reply:
Added